packages/vkui/src/components/AspectRatio/AspectRatio.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { classNames } from '@vkontakte/vkjs';
+import { CSSCustomProperties, HasRootRef } from '../../types';
+import styles from './AspectRatio.module.css';
+
+export interface AspectRatioProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    HasRootRef<HTMLDivElement> {
+  className?: string;
+  /**
+   * По умолчанию, вложенный контент будет растягиваться и заполнять весь блок.
+   */
+  mode?: 'stretch' | 'none';
+  children: React.ReactNode;
+  /**
+   * Например 16 / 9, 4 / 3, 1920 / 1080
+   */
+  ratio: number;
+}
+
+/**
+ * `AspectRatio` позволяет поддерживать постоянное соотношение ширины и высоты.
+ * Его можно использовать для отображения изображений, карт, видео и других медиафайлов.
```

Comment:
```suggestion
 * Его можно использовать для отображения изображений, карт, видео и других медиафайлов.
 
 * @since 5.5.0
 * @see https://vkcom.github.io/VKUI/#/AspectRatio
```